src/app/main.scss

```diff
@@ -422,8 +419,10 @@ a.dim-button {
     will-change: transform;
     z-index: -1;
   }
-  --inventory-column-padding: 24px;
-  --column-padding: calc(2 * var(--inventory-column-padding) - var(--item-margin));
+  @media (min-width: 1079px) {
+    --inventory-column-padding: 24px;
+    --column-padding: calc(2 * var(--inventory-column-padding) - var(--item-margin));
+  }
```

Comment:
Just increase the padding for >= 1080 https://discord.com/channels/316217202766512130/333464468241514497/761107530738434089